src/screens/ChildrenScreen.js

```diff
@@ -1,158 +1,137 @@
+                    />
+                    <Text style={styles.birth}>BIRTH: {u.birth}</Text>
+                  </View>
+                  <View style={styles.lineContainer}>
+                    <Image
+                      style={styles.icon}
+                      source={require("../../assets/icons/injection.png")}
+                    />
+                    <Text style={styles.birth}>LATEST: {u.latest}</Text>
+                  </View>
+                </View>
               </View>
             </View>
           );
         })}
       </View>
-    );
-  };
-  return (
-    <View style={screen.container}>
-      <ScrollView>{renderHistories()}</ScrollView>
-      <Ionicons
-        name="ios-add-circle"
-        size={90}
-        color={colors.lightBlue}
-        style={styles.addIcon}
-        onPress={() =>
-          navigation.navigate("AddChildScreen", { screen: "AddChildScreen" })
-        }
-      ></Ionicons>
+      <View style={styles.foot}>
+        <TouchableOpacity onPress={() => navigation.navigate("AddChildScreen")}>
+          <Image
+            style={styles.addIcon}
+            source={require("../../assets/icons/add-profile.png")}
+          ></Image>
+        </TouchableOpacity>
+      </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
+  mainContainer: {
+    flex: 1,
+  },
+  header: {
+    flex: 0.8,
+  },
+  foot: {
+    flex: 0.2,
+  },
   container: {
-    width: "95%",
-    marginLeft: "5%",
-    marginTop: "2%",
-    paddingBottom: "2%",
-    borderBottomColor: colors.lightBlue_button,
-    borderBottomWidth: 1,
+    height: moderateScale(96),
+    backgroundColor: colors.white,
+    flexDirection: "row",
+    marginBottom: scale(10),
   },
-  listStyle: { paddingRight: "12%" },
-  type: {
-    fontSize: 20,
+  lineContainer: {
+    flexDirection: "row",
+  },
+  name: {
+    fontFamily: "notoSans-regular",
+    fontSize: moderateScale(16),
     fontWeight: "bold",
-    color: colors.lightBlue,
+    lineHeight: moderateScale(22),
+    margin: scale(5),
   },
-  date: {
-    fontSize: 15,
+  birth: {
+    fontFamily: "notoSans-regular",
+    fontSize: moderateScale(12),
+    color: "#1F263E",
+    lineHeight: moderateScale(16.34),
+    margin: scale(5),
   },
-  note: { fontSize: 14 },
-  arrowIcon: {
+  image: {
     flex: 1,
-    alignItems: "center",
-    position: "absolute",
-    marginTop: "6%",
-    right: 0,
+    resizeMode: "center",
+    height: moderateScale(70),
+    width: moderateScale(70),
+    margin: scale(10),
+    justifyContent: "space-around",
+    borderRadius: scale(400 / 2),
+  },
+  body: {
+    flex: 2,
+    justifyContent: "center",
   },
   addIcon: {
-    flex: 1,
-    position: "absolute",
     alignSelf: "flex-end",
-    bottom: 0,
-    right: 0,
+    width: moderateScale(70),
+    height: moderateScale(70),
+    marginRight: moderateScale(23),
+  },
+  icon: {
+    width: moderateScale(10),
```

Comment:
Add 
resizeMode: "stretch"
You can see code in the file ScheduleScreen as a reference